react/components/Share/components/SocialButton.js

```diff
@@ -50,29 +51,23 @@ export default class SocialButton extends Component {
   }
 
   render() {
-    const {
-      url,
-      message,
-      size,
-      socialEnum,
-      buttonClass,
-      iconClass,
-      imageUrl,
-    } = this.props
+    const { url, message, size, socialEnum, buttonClass, iconClass, imageUrl } =
```

Comment:
Oh, that's interesting, our lint did this? Did you install the dependencies on the root folder?

Reply:
Yes, my vs code was complaing about prettier, so i've just installed